tests/pages/jobs/JobCreateForm-cy.js

```diff
@@ -0,0 +1,64 @@
+describe("Job Create Form", function() {
+  beforeEach(function() {
+    cy.configureCluster({
+      jobDetails: true,
+      mesos: "1-for-each-health",
+      nodeHealth: true
+    });
+    cy.visitUrl({ url: "/jobs/overview" });
+  });
+
+  it("displays an error badge for the Services tab", function() {
+    const jobName = "simple";
+    const fullJobName = `${Cypress.env("TEST_UUID")}.${jobName}`;
```

Comment:
This is only for system tests.

Reply:
We use `Cypress.env("TEST_UUID")}` in other jobs integration tests, but I'll drop it here